assets/src/edit-story/components/canvas/karma/backgroundCopyPaste.karma.js

```diff
@@ -0,0 +1,418 @@
+  }
+
+  function getButtonByText(buttonText) {
+    return getElementByQueryAndMatcher('button', getByInnerText(buttonText));
+  }
+
+  function getButtonByAriaLabel(ariaLabel) {
+    return getElementByQueryAndMatcher(
+      'button',
+      getByAttribute('aria-label', ariaLabel)
+    );
+  }
+
+  function getInputByAriaLabel(ariaLabel) {
+    return getElementByQueryAndMatcher(
+      'input',
+      getByAttribute('aria-label', ariaLabel)
+    );
+  }
+
+  function getMediaElement(imageAlt) {
+    return getElementByQueryAndMatcher(
+      '[data-testid="mediaElement"] img',
+      getByAttribute('alt', imageAlt)
+    );
+  }
+
+  function getCanvasElementWrapperById(id) {
+    return fixture.querySelector(
+      `[data-testid="safezone"] [data-element-id="${id}"]`
+    );
+  }
+
+  function getCanvasElementFrameById(id) {
+    return fixture.querySelector(
+      `[data-testid="FramesLayer"] [data-element-id="${id}"]`
+    );
+  }
+
+  async function getBackgroundElementId() {
+    const {
+      state: {
+        currentPage: {
+          elements: [{ id }],
+        },
+      },
+    } = await fixture.renderHook(() => useStory());
+    return id;
+  }
+
+  async function getCanvasBackgroundElementWrapper() {
+    const id = await getBackgroundElementId();
+    return getCanvasElementWrapperById(id);
+  }
+
+  async function getCanvasBackgroundElementFrame() {
+    const id = await getBackgroundElementId();
+    return getCanvasElementFrameById(id);
+  }
+
+  function getPageArea() {
+    return fixture.querySelector(
+      '[data-testid="fullbleed"] [class^="layout__PageAreaWithOverflow"]'
+    );
+  }
+});
+
+const customMatchers = {
+  toBeEmpty: () => ({
+    compare: function (actual) {
+      const innerHTML = actual?.innerHTML ?? '';
+      const pass = innerHTML === '';
+      return {
+        pass,
+        message: pass
+          ? `Expected element to not be empty`
+          : `Expected element to be empty`,
+      };
+    },
+  }),
+  toHaveStyle: (util, customEqualityTesters) => ({
+    compare: function (element, property, expected) {
+      const actual = getComputedStyle(element)[property];
+      const pass = util.equals(actual, expected, customEqualityTesters);
+      return {
+        pass,
+        message: pass
+          ? `Expected element to not have style "${property}: ${expected}"`
+          : `Expected element to have style "${property}: ${expected}" but found "${actual}"`,
+      };
+    },
+  }),
+  toHaveProperty: (util, customEqualityTesters) => ({
+    compare: function (element, property, expected) {
+      const actual = element?.[property] ?? '';
+      const pass =
+        typeof expected === 'string'
+          ? util.equals(actual, expected, customEqualityTesters)
+          : expected.test(actual);
+      return {
+        pass,
+        message: pass
+          ? `Expected element to not have src "${expected}"`
+          : `Expected element to have src "${expected}" but found "${actual}"`,
```

Comment:
Same question here, should the `src` actually be `${property}`?

Reply:
Fixed in 091fad534.